runtime/src/builtin/singletons.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019-2022 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use fvm_shared::address::Address;
+use fvm_shared::ActorID;
+
+lazy_static! {
```

Comment:
This file can also go, these variables are builtin-actor-specific.

Reply:
removed